src/Abp.AutoMapper/AutoMapper/AutoMapExtensions.cs

```diff
@@ -28,5 +34,70 @@ public static TDestination MapTo<TDestination>(this object source)
         {
             return Mapper.Map(source, destination);
         }
+
+        public static void CreateMultiLingualMap<TMultiLingualEntity, TMultiLingualEntityPrimaryKey, TTranslation, TDestination>(
+            this IMapperConfigurationExpression configuration, MultiLingualMapContext multiLingualMapContext)
+            where TTranslation : class, IEntityTranslation<TMultiLingualEntity, TMultiLingualEntityPrimaryKey>
+            where TMultiLingualEntity : IMultiLingualEntity<TTranslation>
+        {
+            configuration.CreateMap<TTranslation, TDestination>();
+            configuration.CreateMap<TMultiLingualEntity, TDestination>().AfterMap((source, destination, context) =>
+            {
+                var translation = source.Translations.FirstOrDefault(pt => pt.Language == CultureInfo.CurrentUICulture.Name);
+                if (translation != null)
+                {
+                    context.Mapper.Map(translation, destination);
+                    return;
+                }
+
+                var defaultLanguage = multiLingualMapContext.SettingManager
+                                                            .GetSettingValue(LocalizationSettingNames.DefaultLanguage);
+
+                translation = source.Translations.FirstOrDefault(pt => pt.Language == defaultLanguage);
+                if (translation != null)
+                {
+                    context.Mapper.Map(translation, destination);
+                    return;
+                }
+
+                translation = source.Translations.FirstOrDefault();
+                if (translation != null)
+                {
+                    context.Mapper.Map(translation, destination);
+                }
+            });
+        }
+
+        public static void CreateMultiLingualMap<TMultiLingualEntity, TTranslation, TDestination>(this IMapperConfigurationExpression configuration, MultiLingualMapContext multiLingualMapContext)
```

Comment:
I think this is shortcut for `int` PK. So, why not re-using the `CreateMultiLingualMap` defined above?